README.MD

```diff
@@ -1,7 +0,0 @@
-# Rapid Coin ICO Smart Contract Audit   
```

Comment:
@yasiriqbal776 please, don't remove the index file of audit

Reply:
@DeRain  Sorry for index file , And i have used openzeppelin via npm package manager.

Reply:
@yasiriqbal776 could you add file again and remove a node_modules directory from the index?

Also please create package.json and add openzeppelin in this file as a dependency.

Reply:
@DeRain i have setup project with package.json and openzepplin import. Please verify it. Thanks